docs/commands.mdx

```diff
@@ -0,0 +1,234 @@
+|-----------------|--------------------------------------------|------------------------------------------------------------|--------------------------------------------------------|----------|
+| `suggestion_id` | The suggestion you want to view voters for | N/A                                                        | `/view voters suggestion_id:lrt97z1s`                  | **true** |
+| `filter`        | The voters to view for this suggestion     | - `All voters` <br /> - `Up voters` <br /> - `Down voters` | `/view voters suggestion_id:lrt97z1s filter:Up voters` | false    |
+
+Down below, you'll find different examples of how suggestions are displayed.
+
+<SuggestionEmbedMessage
+  embed={<ViewVoters filter="upvoters" />}
+  message={{
+    command: {
+      name: '/view voters'
+    },
+    ephemeral: true
+  }}
+  ephemeral
+/>
+
+Running `/view voters suggestion_id:lrt97z1s filter:Up voters` will display all users who have upvoted the suggestion.
+<br />
+
+<SuggestionEmbedMessage
+  embed={<ViewVoters filter="downvoters" />}
+  message={{
+    command: {
+      name: '/view voters'
+    },
+    ephemeral: true
+  }}
+  ephemeral
+/>
+
+Running `/view voters suggestion_id:lrt97z1s filter:Down voters` will display all users who have downvoted the suggestion.
+<br />
+
+<SuggestionEmbedMessage
+  embed={<ViewVoters filter="all" />}
+  message={{
+    command: {
+      name: '/view voters'
+    },
+    ephemeral: true
+  }}
+  ephemeral
+/>
+
+Running `/view voters suggestion_id:lrt97z1s filter:All voters` will display all users who have voted for the suggestion, both upvoters and downvoters.
+
+### User Blocklist Add Command
+
+```
+/user blocklist add <suggestion_id>
+```
+
+This command will allow an administrator to block a specific user from submitting new suggestions. It takes the suggestion ID as input. Read more information on how this works [here] [4].
+
+### User Blocklist Remove Command
+
+```
+/user blocklist remove [suggestion_id|user_id]
+```
+
+This command will allow an administrator to remove a specific user from the suggestion submission blocklist. It takes either a user ID or a suggestion ID as input. Read more information on how this works [here] [4].
+
+| Option          | Description                              | Example                                             | Required |
+|-----------------|------------------------------------------|-----------------------------------------------------|----------|
+| `suggestion_id` | The suggestions ID you wish to reference | `/user blocklist remove suggestion_id:lrt97z1s`     | false    |
+| `user_id`       | The users discord id                     | `/user blocklist remove user_id:158063324699951104` | false    |
+
+#### Footnotes
+- [1]: configuration#images-in-suggestions "Configuration: Images in Suggestions"
+- [2]: anonymous-suggestions.mdx "Anonymous Suggestions"
+- [3]: context-menu#view-up-voters "Context Menu: View Up voters"
+- [4]: blocklist "Suggestions Blocklist"
```

Comment:
I was going to look further into this. I'm not sure why it's not rendering; hence, the removal of the footnotes from the table of contents. I'll do one last look, but I won't be fussed if it doesn't render.

Reply:
Since those are technically link references, not footnotes, they aren't supposed to render in the UI. So I'm updating that section of the document and that should be all set.